src/ngraph/pass/reshape_sinking.cpp

```diff
@@ -313,14 +343,9 @@ static void sink_reshape(shared_ptr<op::Reshape> reshape,
     else
     {
         // combine both reshapes
-        auto new_reshape = combine_reshapes(orig_reshape, reshape);
+        new_reshape = combine_reshapes(orig_reshape, reshape);
+        new_reshape_in_map = new_reshape;
         // remove original reshape now it's combined with a new one
-        // should be safe to remove an already detached node
-        mark_reshape_for_deletion(orig_reshape, reshapes_to_delete);
-        // replace reshape with combined one
-        ngraph::replace_node(reshape, new_reshape);
-        mark_reshape_for_deletion(new_reshape, reshapes_to_delete);
-        write_reshapemap(reorders, new_reshape, new_reshape);
```

Comment:
why are we removing this?

Reply:
missed a call to `replace_reshape();` will add.